hardhat.config.ts

```diff
@@ -70,7 +86,15 @@ const config: HardhatUserConfig = {
     deployer: {
       default: 0,
     },
-    admin: '0x1a00e1e311009e56e3b0b9ed6f86f5ce128a1c01',
+    eoaAdmin: '0x1a00e1E311009E56e3b0B9Ed6F86f5Ce128a1C01',
+    meanDeployer: '0xe48a5173AdE669651120cb5E99e6fE140d4d73da',
+    msig: {
+      default: '0x1a00e1E311009E56e3b0B9Ed6F86f5Ce128a1C01',
```

Comment:
Do we need to have a default? It could be a little error prone

Reply:
While I do agree, we need to set a default for the networks we do not have a multi-sig, and access to the EOA seems like the best approach. Open to suggestions.